.golangci.yml

```diff
@@ -40,6 +40,9 @@ issues:
     - linters:
         - staticcheck
       text: "SA1019:" # silence errors on usage of deprecated funcs
+    - text: "G115: integer overflow conversion"
```

Comment:
This is not strictly necessary currently, but we will need it as soon as we update to a newer version of golangci-lint. We use a newer one in the solidity repo (and 1.57 seems broken on my Mac, so I am running the same from the solidity repo), which turns on G115 by default. We need convert integers constantly, and there is no way to appease the linter (according to issues I found on gh). So if we just add it now, we'll save ourselves some pain when we upgrade golangci-lint.